go/vt/vttablet/tabletserver/query_engine.go

```diff
@@ -176,10 +177,10 @@ func NewQueryEngine(env tabletenv.Env, se *schema.Engine) *QueryEngine {
 	qe.streamConns = connpool.NewPool(env, "StreamConnPool", config.OlapReadPool)
 	qe.consolidatorMode.Set(config.Consolidator)
 	qe.consolidator = sync2.NewConsolidator()
+	qe.streamConsolidator = NewStreamConsolidator(config.ConsolidatorStreamTotalSize, config.ConsolidatorStreamQuerySize, returnStreamResult)
 	if config.ConsolidatorStreamTotalSize > 0 && config.ConsolidatorStreamQuerySize > 0 {
 		log.Info("Stream consolidator is enabled with query size set to %d and total size set to %d.",
 			config.ConsolidatorStreamQuerySize, config.ConsolidatorStreamTotalSize)
-		qe.streamConsolidator = NewStreamConsolidator(config.ConsolidatorStreamTotalSize, config.ConsolidatorStreamQuerySize, returnStreamResult)
```

Comment:
Is there a reason to move this statement out of the conditional check for ConsolidatorStreamTotalSize and ConsolidatorStreamQuerySize

Reply:
No I think this can be reverted, thanks for catching